src/main/java/ai/promoted/java/client/TwoArmExperiment.java

```diff
@@ -0,0 +1,142 @@
+package ai.promoted.java.client;
```

Comment:
I don't know how many classes we plan on having in this package.  Might make sense to move this to a subpackage called .experiment

Also, not sure the value of ".java" in the java domain path.

Reply:
I'm going to move it to delivery.client. I don't think we want it way up at ai.promoted.